packages/expo-doctor/src/checks/ProjectSetupCheck.ts

```diff
@@ -0,0 +1,49 @@
+import fs from 'fs';
+
+import { DoctorCheck, DoctorCheckParams, DoctorCheckResult } from './checks.types';
+
+export class ProjectSetupCheck implements DoctorCheck {
+  description = 'Check for common project setup issues';
+
+  sdkVersionRange = '*';
+
+  async runAsync({ exp, projectRoot }: DoctorCheckParams): Promise<DoctorCheckResult> {
+    const issues: string[] = [];
+
+    // ** possibly-unintentionally-bare check **
+
+    if (
+      (fs.existsSync(`${projectRoot}/ios`) || fs.existsSync(`${projectRoot}/android`)) &&
```

Comment:
I think this still misses the "Are the `/ios` or `/android` folders gitignored"-check.

Right now, if users get presented with this warning, and add it to their gitignore, the warning will still be visible after making that change right?

Also, prefer `path.join` over self-concatenation (usually for Windows):
```suggestion
      (fs.existsSync(path.join(projectRoot, 'ios')) || fs.existsSync(path.join(projectRoot', 'android'))) &&
```